xdsl/dialects/accfg.py

```diff
@@ -341,14 +341,19 @@ def parse_itm() -> tuple[str, SSAValue]:
             attributes = parser.parse_optional_attr_dict()
 
         parser.parse_punctuation(":")
+        pos = parser.pos
```

Comment:
I might be blind but is this used anywhere?

Reply:
last three chars of line 348